doc/classes/TriangleMesh.xml

```diff
@@ -1,11 +1,56 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="TriangleMesh" inherits="RefCounted" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Internal mesh type.
+		Triangle geometry for efficient, physicsless intersection queries.
 	</brief_description>
 	<description>
-		Mesh type used internally for collision calculations.
+		Creates a bounding volume hierarchy (BVH) tree structure around triangle geometry.
+		The triangle BVH tree can be used for efficient intersection queries without involving a physics engine.
+		[b]Performance:[/b] Creating the BVH tree for complex geometry is a slow process and best done in a background thread.
 	</description>
 	<tutorials>
 	</tutorials>
+	<methods>
+		<method name="create_from_faces">
+			<return type="bool" />
+			<param index="0" name="faces" type="PackedVector3Array" />
+			<description>
+				Creates the BVH tree from an array of faces.
+				Each three vertices of the input [param faces] array represent one triangle (face).
+				Returns [code]true[/code] if the tree is build successful.
```

Comment:
Each line break in documentation creates a new paragraph, so try to use line breaks more sparingly to avoid creating too many paragraphs:

```suggestion
				Creates the BVH tree from an array of faces. Each 3 vertices of the input [param faces] array represent one triangle (face).
				Returns [code]true[/code] if the tree is build successful.
```

Also, writing numbers with digits is preferred as it makes them stand out more.